cases/collector/kubernetes_client.go

```diff
@@ -61,6 +61,8 @@ func (c *K8sClientCollector) Do(ctx detek.DetekContext) error {
 		config, err = clientcmd.BuildConfigFromFlags("", kubeconfigPath)
 	}
 
+	config.WarningHandler = rest.NoWarnings{}
```

Comment:
Disable Kubernetes client warning handler.

It remain warning log about api lifecyle on the top of report.html.